distribution/docs/src/main/jdocs/content/_tables/Csw_Registry_Store-table-contents.adoc

```diff
@@ -1,5 +1,9 @@
-:type: documentation
+:title: CSW Registry Store
+:id: Csw_Registry_Store
+:type: table
 :status: published
+:application: ${ddf-registry}
+:summary: Registry CSW Store.
```

Comment:
Not sure. that's how it is in the metatype.xml: https://github.com/codice/ddf/blob/master/catalog/spatial/registry/registry-api-impl/src/main/resources/OSGI-INF/metatype/metatype.xml#L17